src/shared/components/palette/address/write-address-field.html

```diff
@@ -4,20 +4,20 @@
     <h2 class="heading-h2">{{caseField | ccdFieldLabel }}</h2>
 
     <div class="form-group bottom-30" id="postcodeLookup" [ngClass]="{'form-group-error': missingPostcode}" *ngIf="!isExpanded">
```

Comment:
I see other `id` in the html, shouldn't we prefix all of them with id()_?

Reply:
That's a good point, but this can be taken as a separate tech debt ticket, since this links with several fragile unit test cases, and this may have impact in several other places as well.

Reply:
If that wouldn't take a lot of effort, maybe making this occasional improvement within this ticket can save later context switch cost from the person who will pick a separate ticket in that regard.